src/components/Calculator.js

```diff
@@ -7,10 +7,14 @@ import Buttons from "./Buttons";
 
 const Calculator = () => {
   const [value, setValue] = React.useState(0)
+  const operators = ["X", "%", "±", "AC", "+", "-", "/", "="]
+
+  const onButtonClick = (val) => setValue(operators.includes(val) ? '' : val)
```

Comment:
This is setting the value to `''` if `val` is an operator. That means the value will disappear from the screen whenever you click an operator. Is that what happens on an actual calculator?